service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/LagomServiceRouter.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.server;
+
+import play.api.routing.Router;
+
+/**
+ * A Lagom service router.
+ *
+ * This trait doesn't add anything, except that it makes the router created by the LagomServer
```

Comment:
Maybe better to use the term "interface" instead of "trait" here as the target audience are java developers